macros/edr/dbt_artifacts/upload_dbt_invocation.sql

```diff
@@ -106,7 +106,7 @@
       {% do return(orchestrator) %}
     {% endif %}
   {% endfor %}
-  {% do return(none) %}
+  {% do return(elementary.get_first_env_var('DBT_ORCHESTRATOR')) %}
```

Comment:
`get_first_env_var` receives a list rather than a string.

```suggestion
  {% do return(elementary.get_first_env_var(["DBT_ORCHESTRATOR"])) %}
```

Reply:
yeah, an oversight from my part. Fixing that now!